lib/index.js

```diff
@@ -27,6 +27,17 @@ class PWMetrics {
     this.sheets = opts.sheets;
     this.expectations = opts.expectations || false;
     this.tryLighthouseCounter = 0;
+    this.desiredMetrics = [];
+    if (this.flags.desiredMetrics) {
+      this.desiredMetrics = this.flags.desiredMetrics.filter(metric => {
+        if (!Object.values(metrics.ids).includes(metric)) {
+          console.log(getMessageWithPrefix('WARNING', 'UNKOWN_METRIC', metric));
+          return false;
+        }
+        return true;
+      });
+    }
+    this.desiredMetrics = this.desiredMetrics.length ? this.desiredMetrics : ['tti', 'psi', 'ttfmp', 'ttfcp'];
```

Comment:
Consider this code snippet:
```js
var metrics = Object.assign([], ['tti', 'psi', 'ttfmp', 'ttfcp'], ['fv']);
// metrics = [ 'fv', 'psi', 'ttfmp', 'ttfcp' ]
```
 The user only wants the fv metric, if we use that code they'll get unwanted metrics